ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/TestData.kt

```diff
@@ -116,6 +116,115 @@ internal object TestData {
         ),
         assetBaseURL = Constants.assetBaseURL,
     )
+    val template3 = PaywallData(
+        templateName = "3", // TODO-PAYWALLS: use enum
```

Comment:
Since the original data is a string, it might be difficult to use the enum here? Though if we end up moving the enum to the main sdk, then that will be different

Reply:
Yes, that's mainly why I was asking, because I tried using the enum here but realized we need to move the class to the other module